src/main/java/com/easypost/model/Shipment.java

```diff
@@ -552,7 +625,7 @@ public void setTaxIdentifiers(final List<TaxIdentifier> taxIdentifiers) {
 
     /**
      * Get the carrier accounts of this shipment.
-     *
+     * <p>
```

Comment:
This doesn't seem necessary? Accidentally added?

Reply:
Resolved in 28156240aa91125dfeeb163cd5eb6d71869b762a